templates/audit/ubuntu1804cis_rule_4_1_12.rules.j2

```diff
@@ -1,3 +1,6 @@
-{% for proc in priv_procs.stdout_lines -%} 
--a always,exit -F path={{ proc }} -F perm=x -F auid>=1000 -F auid!=4294967295 -k privileged
-{% endfor %}
+-a always,exit -F arch=b32 -S mount -F auid>=1000 -F auid!=4294967295 -k mounts
+-a always,exit -F arch=b32 -S mount -F auid>=1000 -F auid!=4294967295 -k export
+{% if ansible_architecture == 'x86_64' -%} 
+-a always,exit -F arch=b64 -S mount -F auid>=1000 -F auid!=4294967295 -k mounts
+-a always,exit -F arch=b64 -S mount -F auid>=1000 -F auid!=4294967295 -k export
```

Comment:
why the same rule with different tags? I think export is unnecessary

Reply:
You are correct, I removed the exports one. 

(In my defense, I just relabeled them and didn't look at the content, seems it was already like this. :) )

Reply:
I didn't mean to blame you. I only noticed it because I read the new benchmark at this point ;)